lib/fastly_nsq/message_queue_consumer.rb

```diff
@@ -0,0 +1,35 @@
+require 'nsq'
+require_relative 'fake_message_queue'
+
+module MessageQueue
+  class Consumer
+    def initialize(topic:, channel:)
+      @topic = topic
+      @channel = channel
+    end
+
+    def connection
+      consumer.new(params)
+    end
+
+    private
+
+    attr_reader :channel, :topic
+
+    def consumer
+      if ENV['FAKE_QUEUE'].nil?
```

Comment:
http://www.rubydoc.info/docs/rails/4.1.7/ActiveRecord/ConnectionAdapters/Column

ActiveRecord has FALSE_VALUES and TRUE_VALUES... it might be good to check for some of those somewhere or to raise when it this is not set to something unexpected.